### PR TITLE
Enable F29 testing

### PR DESCRIPTION
**Pull request by MartinBasti**
Dropping F27 as it is close to EOL and we don't expect another release
there.

Signed-off-by: Martin Bašti <mbasti@redhat.com>

### CODE REVIEW

**Comment by athos-ribeiro** [comment]
I believe tests will pass now, once you rebase this on mater :)

File: atomic_reactor/plugins/pre_inject_yum_repo.py [hunk 1]
```diff
@@ -72,21 +72,21 @@ def run(self):
...
-        repos_host_cont_mapping = {}
         host_repos_path = os.path.join(self.workflow.builder.df_dir, RELATIVE_REPOS_PATH)
         self.log.info("creating directory for yum repos: %s", host_repos_path)
         os.mkdir(host_repos_path)
 
         for repo, repo_content in self.workflow.files.items():
             yum_repo = YumRepo(repourl=repo, content=repo_content, dst_repos_dir=host_repos_path,
                                add_hash=False)
-            repos_host_cont_mapping[repo] = yum_repo.write_and_return_content()
+            yum_repo.write_content()
```
Comment (athos-ribeiro) [inline atomic_reactor/plugins/pre_inject_yum_repo.py]: nice catch!

File: tests/test_yum_util.py [hunk 2]
```diff
@@ -35,3 +36,15 @@ def test_invalid_config():
...
+
+def test_write_content(tmpdir):
+    test_content = 'test_content'
+    repo = YumRepo(
+        repourl='http://example.com/a/b/c/myrepo.repo', content=test_content,
+        dst_repos_dir=str(tmpdir)
+    )
+    repo.write_content()
+
+    with open(os.path.join(str(tmpdir), repo.filename)) as f:
```
Comment (athos-ribeiro) [inline tests/test_yum_util.py]: note: you could use dst_filename and drop the os dependency here.

Comment (MartinBasti) [reply tests/test_yum_util.py]: yes, however in this case at least the test tests if expected location was used.

**Comment by MartinBasti** [comment]
un-WIP-ed

**Comment by MartinBasti** [comment]
bump

**Comment by MartinBasti** [comment]
No release notes needed, change in travis-ci

**Pull request merged**